lib/pong.js

```diff
@@ -32,6 +33,28 @@ var pong = {
     });
   },
 
+  findPlayers: function(user_names, cb) {
+    var findPlayer = function(user_name) {
+      var deferred = Q.defer();
+      pong.findPlayer(user_name, function (err, player) {
+        if (err) {
+          deferred.reject(err);
+        } else {
+          deferred.resolve(player);
+        }
+      });
+      return deferred.promise;
+    };
```

Comment:
Ah, yeah. If the `findPlayer` function was more like this..

```
  findPlayer: function (user_name, cb) {
    if (user_name[0] == '@') {
      user_name = user_name.substring(1, user_name.length);
    }
    return Player.where({ user_name: user_name }).findOne().exec()
  },
```

That query gets executed and returns a promise. http://mongoosejs.com/docs/api.html#query_Query-exec